wpimath/src/main/java/edu/wpi/first/math/interpolation/InterpolatingDouble.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.interpolation;
+
+/**
+ * A Double that can be interpolated using the InterpolatingTreeMap.
+ *
+ * @see InterpolatingTreeMap
+ */
+public class InterpolatingDouble
+    implements Interpolatable<InterpolatingDouble>,
+        InverseInterpolable<InterpolatingDouble>,
+        Comparable<InterpolatingDouble> {
+  public Double value = 0.0;
```

Comment:
Why Double instead of double?

Reply:
That's how 254 did it

Reply:
Then try `public double value;`. Java autoinits to 0.0.